Rehlate/css/styles.css

```diff
@@ -0,0 +1,757 @@
+}
+.locations .caraousel .caraousel-item p {
+  font-size: 0.75rem;
+  font-weight: 200;
+  line-height: 0.75rem;
+}
+@media screen and (min-width: 75rem) {
+  .locations {
+    margin: 0 3.75rem;
+    padding-top: 6.938rem;
+  }
+  .locations .caraousel {
+    gap: 1.75rem;
+  }
+}
+
+.services {
+  text-align: center;
+  margin: 0 1.25rem;
+}
+.services > img {
+  margin-bottom: -7rem;
+  height: 6.938rem;
+  margin-left: -12rem;
+}
+.services .services-cards {
+  border-radius: 25px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4rem 3rem;
+  gap: 2.5rem;
+  margin-bottom: 4.75rem;
+}
+.services .services-cards .services-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1.25rem;
+}
+.services .services-cards .services-card h5 {
+  font-size: 1.25rem;
+  font-weight: 900;
+  line-height: 1.25rem;
+}
+.services .services-cards .services-card p {
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.75rem;
+  font-family: Lama Sans-thin;
+}
+@media screen and (min-width: 75rem) {
+  .services {
+    margin: 0 3.75rem;
+  }
+  .services .services-cards {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    align-items: center;
+  }
+}
+
+.payment-plans {
+  text-align: right;
+  margin: 0 1.25rem;
+  margin-bottom: 4.75rem;
+}
+.payment-plans hr {
+  border: 1px solid #4C4C4C;
+  margin: 2rem 0;
+}
+.payment-plans #purple-card hr {
+  border: 1px solid #A5A5A5;
+}
+.payment-plans #purple-card button {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: center;
+  align-items: center;
+  font-family: "Lama Sans" !important;
+  gap: 0.75rem;
+  background-color: #46368B;
+  border: 1px solid #fff;
+  border-radius: 15px;
+  padding: 0.75rem 4.5rem;
+  font-size: 1.06rem;
+  font-weight: 300;
+  line-height: 1.5rem;
+  cursor: pointer;
+  color: #fff;
+}
+.payment-plans .plans {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2.5rem;
+}
+.payment-plans .plans .plan {
+  border-radius: 25px;
+  border: 1.5px solid #191919;
+  padding: 2.25rem 1.5rem;
+  box-shadow: 10px 10px 0px 0px #000000;
+}
+.payment-plans .plans .plan h5 {
+  text-wrap: nowrap;
+  font-size: 1.5rem;
+  font-weight: 900;
+  line-height: 1.5rem;
+  margin-bottom: 1rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+}
+.payment-plans .plans .plan h5 span {
+  font-size: 2.25rem;
+  font-weight: 900;
+  line-height: 2.25rem;
+}
+.payment-plans .plans .plan p {
+  font-family: "Lama Sans-thin";
+}
+.payment-plans .plans .plan li + li {
+  margin-top: 0.5rem;
+}
+.payment-plans .plans .plan li {
+  background-image: url(../assets/navigators/check-icon.svg);
+  background-repeat: no-repeat;
+  padding: 0.5rem 0;
+  background-position: center right;
+  padding-right: 1.7rem;
```

Comment:
We can increase the padding-right a little bit; let's say make it **2rem** so that it aligns with the Figma design